### PR TITLE
Extended canvas sample

### PR DESCRIPTION
**Pull request by joyl1216**

### CODE REVIEW

**Comment by Mehul1995** [comment]
Looks good to me! Just that I find the image a bit blurred and less sharp. Does it look polished on the emulator??

**Comment by joyl1216** [comment]
@Mehul1995 Yes, the blurry is because I reduced the size of the screenshot to upload here.

**Comment by Mehul1995** [comment]
Looks good to me then!

File: ExtendedCanvas/app/src/main/java/com/microsoft/device/display/samples/extendedcanvas/ui/Theme.kt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    primaryVariant = teal500,
+    secondary = teal200
+)
+
+private val LightColorPalette = lightColors(
+    primary = teal700,
+    primaryVariant = teal500,
+    secondary = teal200
+
+    /* Other default colors to override
```
Comment (CesarValiente) [inline ExtendedCanvas/app/src/main/java/com/microsoft/device/display/samples/extendedcanvas/ui/Theme.kt:20]: Does it make sense to keep this commented code?

File: ExtendedCanvas/app/src/main/java/com/microsoft/device/display/samples/extendedcanvas/ui/Type.kt [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import androidx.compose.ui.unit.sp
+
+// Set of Material typography styles to start with
+val typography = Typography(
+    body1 = TextStyle(
+        fontFamily = FontFamily.Default,
+        fontWeight = FontWeight.Normal,
+        fontSize = 16.sp
+    )
+    /* Other default text styles to override
```
Comment (CesarValiente) [inline ExtendedCanvas/app/src/main/java/com/microsoft/device/display/samples/extendedcanvas/ui/Type.kt:16]: Does it make sense to keep this commented code?

**Review by CesarValiente — APPROVED** [review]
LGTM! Thanks Joy!

I've added a couple of minor comments about commented code left there, apart of that it looks good!

**Pull request merged**